Sources/Thumbprint/Font.swift

```diff
@@ -81,12 +81,13 @@ public enum Font {
     /// Font with text 3 style that supports scaling for accessibility.
     public static var dynamicText3: UIFont { TextStyle.text3.dynamicFont }
 
-    // MARK: - Testing
-
-    /// Used by snapshot tests to forcefully apply the given trait collection. Do not use in application code.
-    public static var traitCollectionOverrideForTesting: UITraitCollection?
-
     private static var didRegisterFonts = false
+
+    // MARK: - Testing
+    #if DEBUG
```

Comment:
2 things. 

1. After thinking about it, i'm not sure you can specify pre-processor directives on a swift package you import. And if you can i'm not sure how. 
2. You don't use #debug in any of the places this property is called.